examples/npc_only.py

```diff
@@ -22,22 +22,24 @@
 if response.birdview_image is not None:
     file_path = f"{file_name}.jpg"
     response.birdview_image.decode_and_save(file_path)
+
 simulation = iai.BasicCosimulation(
     location=args.location,
     agent_count=10,
     monitor_infractions=True,
-    render_birdview=True,
     ego_agent_mask=[False] * 10,
+    get_birdview=True,
+    random_seed=1,
```

Comment:
I would avoid specifying a random seed in the example, so that if the customer runs it multiple times they get the same result. Anyone wondering whether they can provide a seed can easily check the docs.